### PR TITLE
docs(cpt): review

### PR DESCRIPTION
**Pull request by nerda-codes**

### CODE REVIEW

**Review by bene2k1 — APPROVED** [review]

File: observability/cockpit/troubleshooting/resetting-grafana-password-via-the-api.mdx [hunk 1]
```diff
@@ -6,24 +6,69 @@ meta:
...
 dates:
-  validation: 2023-10-10
+  validation: 2024-04-29
   posted: 2023-04-03
 categories:
   - observability
 ---
 
-You can reset your Grafana password using the following request:
+This page show you how to reset your Grafana user's password through the Scaleway API.
```
Comment (nerda-codes) [inline observability/cockpit/troubleshooting/resetting-grafana-password-via-the-api.mdx:15]:
```suggestion
This page shows you how to reset your Grafana user's password through the Scaleway API.
```

File: observability/cockpit/troubleshooting/resetting-grafana-password-via-the-api.mdx [hunk 2]
```diff
@@ -6,24 +6,69 @@ meta:
...
-  -H "X-Auth-Token: $SCW_SECRET_KEY" \
-  -H "Content-Type: application/json" \
-  -d '{"project_id":"string"}' \
-  "https://api.scaleway.com/cockpit/v1beta1/grafana-users/{grafana_user_id}/reset-password"
-```
+<Macro id="requirements" />
 
-The output displays a new password for your user.
+  - A Scaleway account logged into the [console](https://console.scaleway.com)
+  - [Created](/identity-and-access-management/iam/how-to/create-api-keys/) an API key and that the API key has sufficient [IAM permissions](/identity-and-access-management/iam/reference-content/permission-sets/) to perform the actions described on this page
```
Comment (jcirinosclwy) [inline observability/cockpit/troubleshooting/resetting-grafana-password-via-the-api.mdx:20]:
```suggestion
  - [Created](/identity-and-access-management/iam/how-to/create-api-keys/) an API key with sufficient [IAM permissions](/identity-and-access-management/iam/reference-content/permission-sets/) to perform the actions described on this page
```

File: observability/cockpit/troubleshooting/resetting-grafana-password-via-the-api.mdx [hunk 3]
```diff
@@ -6,24 +6,69 @@ meta:
...
+  - [Installed](https://curl.se/download.html) curl
+
+
+Open a terminal and paste the following command to retrieve your Grafana user ID:
```
Comment (jcirinosclwy) [inline observability/cockpit/troubleshooting/resetting-grafana-password-via-the-api.mdx:24]: Should the steps be numbered, or is it different for Troubleshooting pages?

Comment (nerda-codes) [reply observability/cockpit/troubleshooting/resetting-grafana-password-via-the-api.mdx:24]: This troubleshooting only has two steps and I think we said to only number them if they had 3 or more steps

**Review by jcirinosclwy — APPROVED** [review]

**Pull request merged**